bool IntakeCube
bool MoveToIntakeConfig
float64 time_out
#below config items are for going to position after completion. Could be used for other things too
float64 x
float64 y
bool up_or_down
bool override_pos_limits
float64 dist_tolerance
float64 y_tolerance
float64 x_tolerance
bool hasCube
---
bool timed_out
---
